scripts/utils/trading_strategy_helpers.js

```diff
@@ -237,13 +237,11 @@ module.exports = function(web3 = web3, artifacts = artifacts) {
     assert(targetToken.decimals === 18, "Stable tokens must have 18 decimals")
 
     // Number of brackets is determined by bracketAddresses.length
-    const lowestLimit = targetPrice / (1 + priceRangePercentage / 100)
-    const highestLimit = targetPrice * (1 + priceRangePercentage / 100)
     log(`Lowest-Highest Limit ${lowestLimit}-${highestLimit}`)
```

Comment:
Maybe we don't need this line anymore, since we print the limits in the following log expression?